### PR TITLE
O3-1196: Tablet design (horizontal): "Select local photo instead" button is hidden in attachment popup

### PR DESCRIPTION
**Pull request by 99sumedha**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [x] My work includes tests or is validated by existing tests.

## Summary

Adjust the size of the camera feed area respective to the height of the tablet horizontal view.

## Screenshots

image

## Related Issue

https://issues.openmrs.org/browse/O3-1196

### CODE REVIEW

**Review by brandones — CHANGES_REQUESTED** [review]

File: packages/esm-patient-attachments-app/src/attachments/camera-upload.scss [hunk 1]
```diff
@@ -29,3 +31,9 @@
 .uploadFile {
   display: none;
 }
+
+@media (max-height: $breakpoint-tablet-min){
```
[the remark this answers is not in the document]
Comment (99sumedha) [reply packages/esm-patient-attachments-app/src/attachments/camera-upload.scss:35]: Shall I set it to 600px?

[the remark this answers is not in the document]
Comment (brandones) [reply packages/esm-patient-attachments-app/src/attachments/camera-upload.scss:35]: No, it should just be based on width, not height.

[the remark this answers is not in the document]
Comment (99sumedha) [reply packages/esm-patient-attachments-app/src/attachments/camera-upload.scss:35]: Sure I will make the changes and commit :)

**Comment by 99sumedha** [comment]
@brandones new changes committed :)

**Review by brandones — APPROVED** [review]

**Comment by 99sumedha** [comment]
Thank you :)

**Pull request merged**